pyproject.toml

```diff
@@ -29,6 +29,7 @@ offlinemsmtp = "offlinemsmtp.__main__:main"
 python = "^3.8"
 PyGObject = "^3.42.0"
 watchdog = "^2.1.6"
```

Comment:
we can get rid of this, right?

Reply:
ah, yes, of course! :-)

Reply:
(this got fixed in the latest commit in this PR)